libs/platform/src/lib/dynamic-page/dynamic-page.component.ts

```diff
@@ -164,6 +165,11 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
         this._cd.detectChanges();
     }
 
+    /** @hidden */
+    ngDoCheck(): void {
+        this._cd.markForCheck();
```

Comment:
This looks a bit odd, could you please add a comment with the link to the angular issue, which you included in description?